src/utils/c_utils/src/string_c_utils.c

```diff
@@ -422,50 +422,50 @@ bool MallocAndSubstr(char** target, char* source, size_t len)
  * @param src The source string to be copied.
  * @param destByteLen The size of the destination buffer in bytes, including plus 1 for the null terminator.
  * @param numSrcCharsToCopy The number of source chars to copy. It will be truncated and null-terminated if >= destByteLen.
- * @return int zero on success, or a negative number if there was an error.
+ * @return true on success.
  */
-int ADUC_Safe_StrCopyN(char* dest, const char* src, size_t destByteLen, size_t numSrcCharsToCopy)
+bool ADUC_Safe_StrCopyN(char* dest, const char* src, size_t destByteLen, size_t numSrcCharsToCopy)
```

Comment:
"chars" vs "bytes"?

Reply:
should be named bytes consistently since it could be a char array of utf-8 (or other encoding) encoded chars that each can be up to 4 bytes nowadays (I remember when it was a max of 3).  Will address in follow-up PR.

Reply:
fixed here: https://github.com/Azure/iot-hub-device-update/pull/590